sdk/src/verify.rs

```diff
@@ -245,3 +247,66 @@ where
     invoke_cpi(&account_infos, account_metas, inputs, signer_seeds)?;
     Ok(())
 }
+
+pub fn verify_compressed_accounts<'info, T>(
+    ctx: &Context<
+        '_,
+        '_,
+        '_,
+        'info,
+        impl InvokeAccounts<'info>
+            + LightSystemAccount<'info>
+            + InvokeCpiAccounts<'info>
+            + SignerAccounts<'info>
+            + InvokeCpiContextAccount<'info>
+            + Bumps,
+    >,
+    proof: Option<ProofRpcResult>,
+    light_accounts: &[LightAccount<T>],
```

Comment:
Good idea with the trait, let me try that.

Reply:
Actually doing this from `LightAccountInfo` isn't straightforward because `LightAccountInfo` doesn't know how to deserialize its data which is required to hash it. Better leave it for another pr.